src/App.js

```diff
@@ -157,6 +160,10 @@ const App = ({ locale, location, window, dependencies: { appSessionRef, sessionM
               />
               <PrivateRoute path="/integrations/big-query" exact component={AuthorizationPage} />
               <PrivateRoute path={['/checkout-summary']} exact component={CheckoutSummary} />
+
+              <PrivateRoute path="/menu-demo" exact component={MenuDemo} />
+              <PrivateRoute path={['/editors-demo']} exact component={EditorsDemo} />
```

Comment:
Solo para saber ... El siguiente reemplazo tiene el mismo comportamiento? 
```suggestion
              <PrivateRoute path="/editors-demo" exact component={EditorsDemo} />
```

Reply:
it is the same for this case. But i will put both simplified